CHANGES.md

```diff
@@ -1,3 +1,16 @@
+# Version 1.4.3
+
+## Features
+- Support for the all new bun runtime [#6230](https://github.com/appwrite/appwrite/pull/6230)
+
+## Fixes
+- Add _APP_OPENSSL_KEY_V1 to the compose file [#6199](https://github.com/appwrite/appwrite/pull/6199)
+- Fix: V2 functions env vars [#6215](https://github.com/appwrite/appwrite/pull/6215)
+- Fix: Don't update User Accessed At for Users and Teams APIs [#6222](https://github.com/appwrite/appwrite/pull/6222)
+- Fix: Git deploys with S3 [#6227](https://github.com/appwrite/appwrite/pull/6227)
+- Fix: manual internal id insertion [#6232](https://github.com/appwrite/appwrite/pull/6232)
+- Fix: function timeout [#6235](https://github.com/appwrite/appwrite/pull/6235)
+
```

Comment:
```suggestion
- Ensure collections with datetime attributes are migrated [#6175](https://github.com/appwrite/appwrite/issues/6176)
- Ensure all user data are migrated including status, preferences, and labels [#6174](https://github.com/appwrite/appwrite/issues/6174)
- Allow disabling Custom SMTP Settings [#6203](https://github.com/appwrite/appwrite/issues/6203)
- Fix users pagination in Console [#6178](https://github.com/appwrite/appwrite/issues/6178)
- Fix team memberships migration [#6175](https://github.com/appwrite/appwrite/issues/6175]
- Fix events validation on create/update webhooks [#6218](https://github.com/appwrite/appwrite/issues/6218)
- Fix v2 function execution vars [#6209](https://github.com/appwrite/appwrite/issues/6209)
- Fix create attribute modal null when selecting same time twice [https://github.com/appwrite/console/pull/549](https://github.com/appwrite/console/pull/549)

```

Reply:
Updated my previous comment to:

1. remove console related things
2. remove a duplicate entry
3. point to PRs

Reply:
Lets use this PR https://github.com/appwrite/appwrite/pull/6219 instead of https://github.com/appwrite/appwrite/issues/6218

Reply:
Updated 👌